Makefile

```diff
@@ -6,7 +6,7 @@ IMAGE := $(REGISTRY)/$(PROJECT)
 SRCDIRS := ./cmd ./internal ./apis
 LOCAL_BOOTSTRAP_CONFIG = localenvoyconfig.yaml
 SECURE_LOCAL_BOOTSTRAP_CONFIG = securelocalenvoyconfig.yaml
-ENVOY_IMAGE = docker.io/envoyproxy/envoy:v1.23.1
+ENVOY_IMAGE = docker.io/envoyproxy/envoy:v1.23.0
```

Comment:
looks like a merge conflict, should be v1.23.1 here

Reply:
done